src/content/structured/get-started/install-components.mdx

```diff
@@ -56,3 +56,9 @@ To use the components in a particular framework, follow the framework instructio
   </ic-link>
   .
 </p>
+
+## Using third-party components
+
+If you can't find a suitable component in the Design System, then you should use one from a third-party library. Use the Design System's theming and customisation options to [style](/styles) the third-party component so they look and feel as similar as possible.
+
+Make sure to regulalry check the Design System for new component releases so that you can replace the third-party components once released.
```

Comment:
```suggestion
Make sure to regularly check the Design System for new component releases so that you can replace the third-party components once released.
```